src/main.ts

```diff
@@ -18,7 +18,8 @@ async function runDeployment() {
       "dangerous-clean-slate": optionalBoolean("dangerous-clean-slate", core.getInput("dangerous-clean-slate")),
       "exclude": optionalStringArray("exclude", core.getMultilineInput("exclude")),
       "log-level": optionalLogLevel("log-level", core.getInput("log-level")),
-      "security": optionalSecurity("security", core.getInput("security"))
+      "security": optionalSecurity("security", core.getInput("security")),
+      "timeout": optionalInt("timeout", core.getInput("timeout"))
```

Comment:
can you add `,` at the end?
with this, the next time we add an attribute we don't see a modification in the previous one.